index.html

```diff
@@ -15,14 +15,15 @@
 <!---Header section--->
   <header class="header-section">
   <nav>
-    <div class="logo">My logo</div>
-    <ul>
-      <li><a>Portfolio</a></li>
-      <li><a>About</a></li>
-      <li>Contact</li>
+    <a class="logo" href="#">My logo</a>
+    <ul id="list-items">
+      <li><a href="#">Portfolio</a></li>
+      <li><a href="#">About</a></li>
+      <li><a href="#">Contact</a></li>
     </ul>
+   <img src="./images/union-img.png" alt="union icon">
   </nav>  
-  <div class="bio">
+  <section class="bio">
     <h1 class="heading">
       I’m Diana<br>
```

Comment:
- [Optional] DEar @DianaBeki, Good job making the changes. I'm impressed. But I can see you have wrap all your section with the `heade` tag. Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.  🛑 This is important for you to implement: I strongly suggest you make the change.